### DISCUSSION TITLE
What is the event for commit on specific file ?

### DISCUSSION THREAD

**Discussion by solidsnail** [community/community#25857] [Actions]
<p>Hi there,</p>
<p>I wanna trigger the workflow only if a specific file has been changed after pushing.</p>
<p>How could I achieve this ?</p>
<p>Thank you in advance </p>

**Comment by tuler** [comment, marked as answer]
<p>Use <code>paths</code> filter:</p>
<p><a href="https://help.github.com/en/articles/workflow-syntax-for-github-actions#onpushpull_requestpaths" rel="nofollow noopener">https://help.github.com/en/articles/workflow-syntax-for-github-actions#onpushpull_requestpaths</a></p>
